src/main/java/seedu/address/logic/parser/ListRegisteredPersonsCommandParser.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.ListRegisteredPersonsCommand;
+
+/**
+ * Parses input arguments and creates a new ListRegisteredPersonsCommand object
+ */
+public class ListAttendeesCommandParser implements Parser<ListRegisteredPersonsCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ListRegisteredPersonsCommand
+     * and returns an ListRegisteredPersonsCommand object for execution.
+     */
+    public ListRegisteredPersonsCommand parse(String args) {
```

Comment:
The class names are different, that is why Travis is failing

Reply:
Alright, thanks! Missed out some commits earlier.